src/ui/hooks/useCodePushVersion.test.ts

```diff
@@ -0,0 +1,35 @@
+import CodePush, { LocalPackage } from 'react-native-code-push'
+
+import { eventMonitoring } from 'libs/monitoring'
+import { renderHook, waitFor } from 'tests/utils'
+
+import { useCodePushVersion } from './useCodePushVersion'
+
+describe('useCodePushVersion', () => {
+  it('returns empty string if CodePush.getUpdateMetadata returns null', async () => {
```

Comment:
`should return empty string when there are not CodePush information`
C'est un peu moins technique